plugins/lookup/conjur_variable.py

```diff
@@ -102,6 +102,7 @@
 from ansible.module_utils.urls import open_url
 from ansible.utils.display import Display
 import ssl
+from pathlib import Path
```

Comment:
This import is not used in these changes

Reply:
done